_posts/2022-10-04-clicky-reflection-alm7468.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+author: alm7468
+title: “April’s Clicky Reflection”
```

Comment:
These `"` might be smartquotes. Did you paste from a word processing program?

I'm not sure it'll be a problem, but if there are any problems, this is a place to look.

Reply:
I swapped the quotes out, thank you!